packages/css/src/ordered-list/ordered-list.scss

```diff
@@ -5,39 +5,55 @@
 
 @import "../../utils/breakpoint";
 
+/**
+ * - Applies border box sizing.
+ * - Resets whitespace and list styles.
+ */
 @mixin reset {
   box-sizing: border-box;
+  list-style-type: none;
   margin-block: 0;
+  padding-inline-start: 0;
   -webkit-text-size-adjust: 100%;
 }
 
+/** A list is a grid column with a gap between items. */
 .amsterdam-ordered-list {
+  @include reset;
+
+  display: grid;
+  gap: var(--amsterdam-ordered-list-gap);
+}
+
+/**
+ * A list with markers:
+ * - sets typography for its text content
+ * - applies correct spacing at the start of items.
+ */
+.amsterdam-ordered-list:not(.amsterdam-ordered-list--no-markers) {
   color: var(--amsterdam-ordered-list-color);
-  display: flex;
-  flex-direction: column;
   font-family: var(--amsterdam-ordered-list-font-family);
   font-size: var(--amsterdam-ordered-list-narrow-font-size);
   font-weight: var(--amsterdam-ordered-list-font-weight);
-  gap: var(--amsterdam-ordered-list-gap);
   line-height: var(--amsterdam-ordered-list-line-height);
+  list-style-type: var(--amsterdam-ordered-list-list-style-type);
 
   @media screen and (width > $amsterdam-breakpoint) {
     font-size: var(--amsterdam-ordered-list-wide-font-size);
   }
 
-  @include reset;
-}
-
-.amsterdam-ordered-list__item {
-  padding-inline: var(--amsterdam-ordered-list-item-padding-inline);
+  > .amsterdam-ordered-list__item {
```

Comment:
Is the child combinator (`>`) used here to prevent this styling from seeping through to nested list list items? Because that doesn't work, a nested list has the exact same structure as a regular list. Both regular list list items and nested list list items get this styling now.

Reply:
Yes it is. And to prevent unordered list items being affected by ordered list item styles and vice versa.

Nested list items get these styles applied through their own list parent, or am I missing something?

Reply:
> to prevent unordered list items being affected by ordered list item styles and vice versa

But that's already taken care of by the BEM classes, right? Ordered list items get `.amsterdam-ordered-list__item`, unordered list items get `.amsterdam-unordered-list__item`. 

> Nested list items get these styles applied through their own list parent

Yes, exactly. I guess my point is: currently `> .amsterdam-ordered-list__item` does the exact same thing as `.amsterdam-ordered-list__item`, so why do we need the child combinator? 

For nested lists, the `li` style is currently first set by lines 46-47, and then overwritten by 56-67.

Reply:
Sharp as a knife, you! We don’t even need the parent selector.

Reply:
I’ve removed the child combinator.

And your mention of overriding styles made me realise that we didn’t indent unordered list items nested in an ordered list and vice versa. Now we do.